### PR TITLE
Add robot time to `robot_status_t`

### PR DESCRIPTION
**Pull request by gavincangan**

### CODE REVIEW

File: include/lcmtypes/robot_msgs/robot_status_t.hpp [hunk 1]
```diff
@@ -776,12 +788,13 @@ int robot_status_t::_getEncodedSizeNoHash() const
...
     enc_size += __int64_t_encoded_array_size(NULL, 1);
     enc_size += __double_encoded_array_size(NULL, 1);
+    enc_size += __double_encoded_array_size(NULL, 1);
     return enc_size;
 }
 
 uint64_t robot_status_t::_computeHash(const __lcm_hash_ptr *)
 {
-    uint64_t hash = 0xed8c186e0dc93b9cLL;
+    uint64_t hash = 0x7c777ba6462911cdLL;
```
Comment (dmsj) [inline include/lcmtypes/robot_msgs/robot_status_t.hpp:797]: where does this value come from?

**Review by dmsj — COMMENTED** [review]
does this change any of our existing data processing code?

File: include/lcmtypes/robot_msgs/robot_status_t.hpp (hunk 1 above)
Comment (5yler) [reply include/lcmtypes/robot_msgs/robot_status_t.hpp:797]: Autogenerated by `lcm-gen`

**Review by 5yler — APPROVED** [review]
LGTM platform & style

**Review by duanyutong — APPROVED** [review]

**Comment by gavincangan** [comment]
> does this change any of our existing data processing code?

It does not change our data processing code

**Pull request merged**